packages/iTwinUI-react/package.json

```diff
@@ -47,16 +47,16 @@
   ],
   "scripts": {
     "build": "yarn clean:build && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
-    "build:watch": "yarn clean:build && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
-    "clean:build": "yarn clean:coverage && rimraf esm && rimraf cjs",
+    "build:watch": "concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
+    "clean:build": "rimraf esm && rimraf cjs",
```

Comment:
Just curious, why are we not running `yarn clean:coverage` anymore? Also, what is stored in `packages/iTwinUI-react/coverage`? Because I could not find this folder locally

Reply:
`coverage/` is produced when running `yarn test --coverage`. i removed it because it has nothing to do with build